quarkchain/tools/archivesnapshot.sh

```diff
@@ -0,0 +1,20 @@
+sshpass -p $PriKey scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null u446960@u446960.your-storagebox.de:public-data/VERSION VERSION
+
+filename=`cat VERSION`
+
+if [ $filename = "Archive-A.tar.gz" ]; then
+  filename="Archive-B.tar.gz"
+else
+  filename="Archive-A.tar.gz"
+fi
+echo $filename
+
+
+rm $filename
+curl https://s3-us-west-2.amazonaws.com/qkcmainnet/data/`curl https://s3-us-west-2.amazonaws.com/qkcmainnet/data/LATEST`.tar.gz --output $filename
```

Comment:
But we still need s3 in here, why?

Reply:
S3 is used by all users, and storage box only for dev internal use.